OmarEspinosaCastro/30-Agosto/ArrayList.java

```diff
@@ -0,0 +1,33 @@
+
+package collection;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArrayList {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        int numberLines = in.nextInt();
+        List<List<Integer>> lineas = new ArrayList<>();
```

Comment:
English, improve the name. example: rowListList

Reply:
Variable name was changed